### PR TITLE
pybricks.common: Charger, IMU and System autocompletion

### PR DESCRIPTION
**Pull request by lobodpav**
Added missing API auto-completion Charger and System classes.

### CODE REVIEW

File: src/pybricks/hubs.pyi [hunk 1]
```diff
@@ -17,17 +17,20 @@ class EV3Brick:
...
 
 
 class TechnicHub:
     def __init__(self, top_size: Axis, front_side: Axis): ...
     battery: Battery
     light: ColorLight
+    system: System
 
 
 class PrimeHub:
```
Comment (dlech) [inline src/pybricks/hubs.pyi:36]: It looks like we also need to add `EssentialHub` to this files.

Comment (dlech) [reply src/pybricks/hubs.pyi:36]: It has the following

```c
typedef struct _hubs_EssentialHub_obj_t {
    mp_obj_base_t base;
    mp_obj_t battery;
    mp_obj_t buttons;
    mp_obj_t charger;
    mp_obj_t imu;
    mp_obj_t light;
    mp_obj_t system;
} hubs_EssentialHub_obj_t;
```

Comment (dlech) [reply src/pybricks/hubs.pyi:36]: `imu` is also missing from all hubs that support it.

Comment (lobodpav) [reply src/pybricks/hubs.pyi:36]: How about a separate PR for these?

Also, `MoveHub` assigns `_SimpleAccelerometer()` to the `imu` property whereas `TechnicHub` and `PrimeHub` assign the `_IMU()` instance. A little weird that these two have such a different class names.

Comment (dlech) [reply src/pybricks/hubs.pyi:36]: Sure. The move hub literally just has an accelerometer whereas the other hubs have an IMU with accelerometer and gyro, thus the different classes.

Comment (lobodpav) [reply src/pybricks/hubs.pyi:38]: Thanks for explaining. Added `IMU` to supported Hubs. Will add the `EssentialHub` in a separate PR.

**Comment by dlech** [comment]
LGTM. Can we add a changelog entry?

**Comment by lobodpav** [comment]
> LGTM. Can we add a changelog entry?

Done.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,9 @@
 
 ## Unreleased
 
+### Added
+- Code auto-completion for Hubs' `Charger`, `IMU` and `System`
```
[the remark this answers is not in the document]
Comment (dlech) [reply CHANGELOG.md:8]: Or maybe I would be less confused if it said `hub.charger`, `hub.imu` and `hub.system`.

[the remark this answers is not in the document]
Comment (lobodpav) [reply CHANGELOG.md:8]: I meant `Hubs'` as a possesive plural. Will use the `hub.` prefix instead.

File: src/pybricks/_common.pyi [hunk 3]
```diff
@@ -7,6 +7,12 @@ from .geometry import Axis, Matrix, vector
 from .parameters import Button, Color, Direction, Side, Stop, Port
 
 
+class System:
+    def set_stop_button(self, button: Button): ...
+    def shutdown(self): ...
```
[the remark this answers is not in the document]
Comment (lobodpav) [reply src/pybricks/_common.pyi:12]: I see. Thanks, will fix it.

File: CHANGELOG.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (lobodpav) [reply CHANGELOG.md]: Done.

**Pull request merged**